openmdao/docs/features/recording/case_reader.rst

```diff
@@ -0,0 +1,161 @@
+matter what case recorder was used.
+Currently, OpenMDAO only supports the :class:`SqliteCaseReader<openmdao.recorders.sqlite_reader.SqliteCaseReader>` case
+recorder. Therefore, all the examples will
+make use of this recorder. OpenMDAO will support other case recorders in the future.
+
+.. automethod:: openmdao.recorders.sqlite_reader.SqliteCaseReader.__init__
+    :noindex:
+
+Determining What Sources and Variables Were Recorded
+----------------------------------------------------
+
+The :code:`CaseReader` object provides methods to determine which objects in the original problem were responsible
+for recording cases and what variables they recorded.
+
+The :code:`list_sources` method provides a
+list of the names of objects that are the recording sources of the recorded data in the file.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_sources
+    :noindex:
+
+The complementary :code:`list_source_vars` method will provide a dictionary of the input and output variables recorded
+by each of those sources.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_source_vars
+    :noindex:
+
+Here is an example of their usage.
+
+.. embed-code::
+    openmdao.recorders.tests.test_sqlite_reader.TestFeatureSqliteReader.test_feature_list_sources
+    :layout: interleave
+
+
+Case Names
+----------
+
+A :code:`CaseReader` object contains a collection of :class:`Case<openmdao.recorders.case.Case>`
+objects, one for each individual recording of an object
+which can be anyone of these objects: driver, problem, component, or solver.
+
+:code:`Case` objects are uniquely identified in a case recorder file by their case names. A Case name is a string.
+As an example, here is a case name:
+
+    'rank0:ScipyOptimize_SLSQP|1|root._solve_nonlinear|1'
+
+The first part of the case name indicates which rank or process that the case was recorded from. The remainder of the
+case name shows the hierarchical path to the object that was recorded along with the iteration counts for each object
+along the path. It follows a pattern of repeated pairs of
+
+    - object name ( problem, driver, system, or solver )
+    - iteration count
+
+These are separated by the :code:`|` character.
+
+So in the given example, the case is:
+
+    - from rank 0
+    - the first iteration of the driver, `ScipyOptimize_SLSQP`
+    - the first execution of the `root` system which is the top-level model
+
+Getting Names of the Cases
+--------------------------
+
+The :code:`list_cases` method returns the names of the cases in the order in which
+the cases were executed. You can optionally request cases only from a specific :code:`source`.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_cases
+    :noindex:
+
+.. _list_cases_args:
+
+There are two optional arguments to the :code:`list_cases` method that effect what is returned.
```

Comment:
I think you want `affect` here rather than `effect`

As a verb `effect` means "cause something to happen", while `affect` means "to have an effect on"